content/may-2016-meet-minutes.md

```diff
@@ -6,30 +6,30 @@ Tags: Meeting Minutes
 After a long time we were back to IMSc Ramanujam auditorium.  The
 meetup Intro was given by Abhishek.
 
-We started with second talk since Shreyas was not available for this
+We started with second talk since Shrayas was not available for this
 meetup.
 
 ## Introduction to Celery
 
 The first talk was given by Shanmuga. He started explaining how and
-why he started using celery in his project. He continued with
-requirement for setting up celery like choosing a broker (RabbitMq or
+why he started using Celery in his project. He continued with
+requirement for setting up Celery like choosing a broker (RabbitMq or
 Redis etc). He explained the advantages and disadvantages of those
-brokers. Then he started explaining a prototype written by him.  I am
-not available after some 4 slides since myself and vengatesh left to
+brokers. Then he started explaining a prototype written by him.  I was
+not available after some 4 slides since myself and Vengatesh left to
 alert tea for networking tea.
```

Comment:
"alert tea for networking tea." could be better phrased as "check the status of snacks/tea for the networking break ahead."